WIP-CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - Bumped backup-postgres image to use tag `1.2.0`, which includes newer versions of the postgresql client
 - Fixed Test User RBAC
 - Fixed the shifted comment after running init
+- Fluentd-plugins was changed from using "fluent-plugin-kubernetes_metadata_filter: 3.1.0" to "fluent-plugin-kubernetes_metadata_filter: 2.13.0" and "fluent-plugin-elasticsearch: 5.2.3" to "fluent-plugin-elasticsearch: 5.2.4". These two changes was done to fix issues regarding mapper_parsing_exceptions in our latest apps-version
```

Comment:
But the changes in `fluent-plugin-elasticsearch` has nothing to with this as we are not using that plugin right? We are using `fluent-plugin-opensearch` if I'm not mistaken.